app/templates/components/goodcity/add-item-overlay.hbs

```diff
@@ -0,0 +1,95 @@
+<div class="add-item small-9 columns" align="center">
+  {{#popup-overlay open=open}}
+  <section class="main-section move-item-to-box">
+    <div class="row">
+      <div class="small-12 columns heading">
+        <span>{{t 'box_pallet.move_to'}} {{entity.storageTypeName}} {{entity.inventoryNumber}}.</span>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="columns small-12">
+        <div class="bottom-header">
+          <div class="tab-container">
+            <div class="tab small">
+              <div class="icon-holder">{{fa-icon 'tag' }}</div>
+              <div class="text-holder">{{pkg.inventoryNumber}}</div>
+            </div>
+            <div class="tab large">
+              <div class="icon-holder">{{fa-icon 'map-marker-alt' }}</div>
+              <div class="text-holder">{{pkgLocationName}}</div>
+            </div>
+            <div class="tab end large">
+              <div class="sub-tabs">
+                <div class="icon-holder">{{fa-icon 'paper-plane' }}</div>
+                <div class="text-holder">{{pkg.totalDispatchedQty}}</div>
+              </div>
+              <div class="sub-tabs">
+                <div class="icon-holder">{{fa-icon 'shopping-basket' }}</div>
+                <div class="text-holder">{{pkg.totalDesignatedQty }}</div>
+              </div>
+              <div class="sub-tabs">
+                <div class="icon-holder">{{fa-icon 'eye' }}</div>
+                <div class="text-holder">{{pkg.availableQty}}</div>
+              </div>
+            </div>
+          </div>
+          <div class="item-summary">
+            <div class="content item-img">
+              <img src={{pkg.thumbImageUrl}} class="thumb" />
+            </div>
+            <div class="content item-description">
+              <div>
+                {{pkg.notes}}
+              </div>
+              <div>
+                {{pkg.notes}}
+              </div>
```

Comment:
Are we intentionally showing `notes` text twice?

Reply:
It is added as per the mockup @swatijadhav .